tensorflow/python/ops/nn_ops.py

```diff
@@ -3197,12 +3202,14 @@ def max_pool_with_argmax_v2(input,
   ksize = _get_sequence(ksize, 2, 3, "ksize")
   strides = _get_sequence(strides, 2, 3, "strides")
 
-  return gen_nn_ops.max_pool_with_argmax(input=input,
-                                         ksize=ksize,
-                                         strides=strides,
-                                         padding=padding,
-                                         Targmax=output_dtype,
-                                         name=name)
+  return gen_nn_ops.max_pool_with_argmax(
+      input=input,
+      ksize=ksize,
+      strides=strides,
+      padding=padding,
+      Targmax=output_dtype,
+      include_batch_in_index=include_batch_in_index,
```

Comment:
@alextp @reedwm  we don't need to add the argument to V1 function below: max_pool_with_argmax_v1, right?

Reply:
I think it should be added. But @alextp just added the V1 function so he would know better.